.gitignore

```diff
@@ -9,3 +9,4 @@ __pycache__/
 tests/_docs/api
 tests/_docs/build
 .tox/
+.venv/
```

Comment:
```suggestion
```
I assume this is your own venv and not tool created? This would best be placed in a personal .gitignore file.